website/mails/mails.py

```diff
@@ -483,25 +483,25 @@ def get_english_article(word):
 
 REGISTRATION_BULK_UPLOAD_PRODUCT_OWNER = Mail(
     'registration_bulk_upload_product_owner',
-    subject='Here goes the email subject'
+    subject='Registry Could Not Bulk Upload Registrations'
 )
 
-REGISTRATION_BULK_UPLOAD_INITIATOR_SUCCESS = Mail(
-    'registration_bulk_upload_initiator_success',
-    subject='Here goes the email subject'
+REGISTRATION_BULK_UPLOAD_SUCCESS_ALL = Mail(
+    'registration_bulk_upload_initiator_success_all',
```

Comment:
This does not match the actual template name.

Reply:
Done/fixed